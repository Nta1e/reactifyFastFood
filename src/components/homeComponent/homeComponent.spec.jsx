import React from 'react';
import { shallow } from 'enzyme';
import LandingPage from './homeComponent';

describe('test landing page component', ()=>{
    let wrapper;
    beforeEach(()=>{
        wrapper = shallow(<LandingPage />)
    });
    it('it mounts', ()=>{
        expect(wrapper).toMatchSnapshot();
    })
});