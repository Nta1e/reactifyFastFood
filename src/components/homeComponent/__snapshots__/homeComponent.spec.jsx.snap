// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test landing page component it mounts 1`] = `ShallowWrapper {}`;
